Wrappers/Python/cil/utilities/dataexample.py

```diff
@@ -46,19 +45,13 @@ def get(cls, size=None, scale=(0,1), **kwargs):
 class REMOTEDATA(DATA):
 
     FOLDER = ''
-    URL = ''
-    FILE_SIZE = ''
+    ZENODO_RECORD = ''
+    ZIP_FILE = ''
 
     @classmethod
     def get(cls, data_dir):
         return None
 
-    @classmethod
-    def _download_and_extract_from_url(cls, data_dir):
-        with urlopen(cls.URL) as response:
-            with BytesIO(response.read()) as bytes, ZipFile(bytes) as zipfile:
-                zipfile.extractall(path = data_dir)
-
     @classmethod
     def download_data(cls, data_dir):
```

Comment:
maybe only ask for `input` if `prompt`?

```suggestion
    def download_data(cls, data_dir, prompt=True):
```